CONTRIBUTING.md

```diff
@@ -1,19 +1,58 @@
 ## Statuttendringer
-Alle statuttendringer må godtas på generalforsamling. Unntaket gjelder endringer
-som ikke endrer betydningen av statuttene, som beskrevet i § 8.3.4. Dette betyr 
-at alle pull-requests som inneholder endringer av betydningen til statuttene må 
-godkjennes på generalforsamling før de kan merges. Andre endringer kan merges av 
-en i Hovedstyret eller etter godkjenning fra Hovedstyret.
+Alle statuttendringer må godtas på generalforsamling. Unntaket gjelder
+endringer som ikke endrer betydningen av statuttene, som beskrevet i § 8.3.4.
+Dette betyr at alle pull-requests som inneholder endringer av betydningen til
+statuttene må godkjennes på generalforsamling før de kan merges. Andre
+endringer kan merges av en i Hovedstyret eller etter godkjenning fra
+Hovedstyret.
 
 >**§ 8.3.4.**   
-Hovedstyret har til enhver tid myndighet til å endre oppsett, utforming og 
-formuleringer samt rette skrivefeil i Abakus sine statutter såfremt disse 
-ikke endrer innholdet og betydningen i de aktuelle statuttene. Eventuelle 
+Hovedstyret har til enhver tid myndighet til å endre oppsett, utforming og
+formuleringer samt rette skrivefeil i Abakus sine statutter såfremt disse ikke
+endrer innholdet og betydningen i de aktuelle statuttene. Eventuelle
 endringsforslag må offentliggjøres 1. februar eller 1. september.
 
 > Hovedstyret må redegjøre for de endringer de har gjort ved å offentliggjøre 
-disse på foreningens nettside. Dersom et medlem av Abakus gir skriftlig uttrykk 
-til Hovedstyret innen to -2- uker om at en endring ikke ivaretar statuttens 
-opprinnelige intensjon må endringen behandles på generalforsamling. Endringer 
-som ikke blir disputert i løpet av denne perioden trer i kraft to -2- uker etter 
-offentliggjøringen.
+disse på foreningens nettside. Dersom et medlem av Abakus gir skriftlig uttrykk
+til Hovedstyret innen to -2- uker om at en endring ikke ivaretar statuttens
+opprinnelige intensjon må endringen behandles på generalforsamling. Endringer
+som ikke blir disputert i løpet av denne perioden trer i kraft to -2- uker
+etter offentliggjøringen.
+
+## Innsending av forslag til endringer
+
+Det er to måter å sende inn forslag på det beste er å lage en pull-request på
```

Comment:
Den første setningen kunne kanskje trengt et komma etter "Det er to måter å sende inn forslag på".